ldappool/__init__.py

```
@@ -0,0 +1,450 @@
+        with self.get() as conn:
+            try:
+                return True
+            except Exception as ldaperr:
+                try:
+                    if conn.search_s("cn=config", ldap.SCOPE_ONELEVEL) != []:
+                        return True
+                    else:
+                        # we might have ACI's in place
+                        return True
+                except Exception as ldaperr:  # collect with parent exception
+                    pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)